src/app/clients/edit-client/edit-client.component.html

```diff
@@ -84,9 +84,21 @@
             <input matInput formControlName="externalId">
           </mat-form-field>
 
+          <mat-checkbox *ngIf="editClientForm.value.legalFormId === 1" fxFlex="48%" labelPosition="before" formControlName="isStaff" class="margin-v">
```

Comment:
Is this related to the email field?

Reply:
It is just to set a better view distribution, I mean set the contact data, mobile and email in the same line